src/common/src/hash/key.rs

```diff
@@ -235,11 +238,17 @@ pub struct FixedSizeKey<const N: usize> {
 /// See [`crate::hash::calc_hash_key_kind`]
 #[derive(Clone, Debug)]
 pub struct SerializedKey {
-    key: Vec<Datum>,
+    key: Row,
```

Comment:
(off the topic of this PR) Why was the struct named `SerializedKey`? It's actually `Vec<Datum>`...  @liurenjie1024

Reply:
It was meant to serialize keys using some serialized format, and `Vec<Datum>` was just a temporary solution.